subsys/dfu/img_util/flash_img.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SYS_LOG_DOMAIN "fota/flash_block"
+#define SYS_LOG_LEVEL SYS_LOG_LEVEL_ERROR
```

Comment:
Sorry I was unclear in my earlier review.

What Zephyr usually does is to define a CONFIG_SYS_LOG_FOO_LEVEL (see for example CONFIG_SYS_LOG_CLOCK_CONTROL_LEVEL, CONFIG_SYS_LOG_SENSOR_LEVEL, CONFIG_SYS_LOG_GPIO_LEVEL) for each "FOO" which wants to log something.

To do the same thing, you can make a new Kconfig option named SYS_LOG_MCUBOOT_LEVEL, which defaults to the error value, then write `#define SYS_LOG_LEVEL CONFIG_SYS_LOG_MCUBOOT_LEVEL` here and in any other files in this subsystem that need to log.

That will let the users change the log level without editing all the sources, just by making changes to their project configuration file.